tests/ert_tests/status/test_utils.py

```diff
@@ -28,52 +27,3 @@ def test_format_running_time(self):
 
         for t in tests:
             self.assertEqual(t["expected"], format_running_time(t["seconds"]))
-
-    def test_calculate_progress(self):
```

Comment:
Are these not possible to test?

Reply:
As `_calculate_progress` function was removed I guess there is no need of having this test anymore :shrug:

Reply:
Thanks. For reference https://github.com/equinor/ert/pull/2960/files#diff-0735d41fdd807327ac2ea6450664124e2b521281a613966c2254a8d33b887b25R194

Reply:
Found a bug for calculation of `_progress()` https://github.com/equinor/ert/issues/3092

I suggest we resolve this outside of this PR.
This was found because a test now needs to ensure that total realizations are also fetched correctly.
This was not the case before, as only the maths given correct numbers were tested.